.github/workflows/atd_moped_build_stage.yml

```diff
@@ -90,40 +90,6 @@ jobs:
           source $(pwd)/.github/workflows/aws-heroku-helper.sh
```

Comment:
I'm actually wondering if we should remove this file entirely. This is the github action to trigger the old moped test deployment. It will not work because the heroku script was removed a while back https://github.com/cityofaustin/atd-moped/pull/781